### ISSUE TITLE
a11y: Fix textual descriptions for cluster appearance in multiple places

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12807]
a11y Issues: 138, 139, 144

**Issue 138 - Textual description missing for interactive image**

The textual description is not specified for the interactive images that resulted in screen reader users finding it difficulty to understand the purpose of the interactive images.

Occurrences:

Cluster Appearance
- "Information" icon associated with "Use custom badge" checkbox

DaemonSet: Create
- "Information" icon associated with Pull Secrets input field) 

Repository: Create 
- "Information" icon associated with "Key" label)

Users Retention Settings
- "Information" icon associated with "Inactivity period" and "User retention process schedule input field" input fields
- "Warning" icon present in input field)

Import Extension Catalog
- "Information" icon associated with "Image Pull Secrets" text

Banners
- "Information" icon associated with "Show Login Consent as a modal dialog" checkbox

Example:

![image](https://github.com/user-attachments/assets/9b784d08-27f1-4247-b444-d1bfb5fd8b1a)

Remediation:

Provide hidden textual description as "Information - Use custom badge" within <i> element. Position the text off-screen via CSS.

Alternatively, Apply the following changes:
• Provide role="img" for <i> element containing mentioned informative image.
• Provide textual description such as "Information" using "aria-label" attribute.

**Issue 139 - No textual description provided for SVG element**

The SVG image lacks textual description that resulted in screen reader users unable to access the information conveyed through the image.

Occurs: Main Content, Images in SIDE NAVIGATION and RANCHER HEADER sections

![image](https://github.com/user-attachments/assets/60f1d02a-8303-466c-8c21-08814ad7be41)

Add textual description of the image by using the <title> element within <svg> element. Ensure that the textual description is descriptive of image's purpose.

**Issue 144 - Additional content available on hover not hoverable and persistent**

Additional content that becomes available when users hover over with a mouse on the image does not stay persistent. 
Moreover, when low vision users try to move their mouse on the additional content in order to read it, the content disappears resulting in users with low vision found it difficult to access the additional content.

Occurrences:

Cluster Appearance
- Overwrites the default cluster name abbreviation. (which gets available when users hover on "Information" icon associated with "Use custom badge" checkbox)

Repository: Create 
- Paste lines of Key=Value or... (which gets available on activating "Add Label and "Add Annotation" buttons > hover on "Information" icon associated with "Key" label)

Users Retention Settings
- User duration units... and The user retention process... (which gets available on activating checkboxes such as "Disable user accounts after..." and "Delete user accounts after..." checkboxes > hover on "Information" icon associated with "Inactivity period" and "User retention process schedule input field" input fields)
- User retention process... (which gets available on activating checkboxes such as "Disable user accounts after..." and "Delete user accounts after..." checkboxes > type in "User retention process schedule" input field > hover on "Warning" icon present in input field)

Cluster Dashboard
- 10/29/2024 21:33:16 (which gets available on hovering Created: 12 days ago text)
- Customize Appearance (which gets available on hovering Cluster Tools interactive image)

Import YAML
- "end of the stream or a document…" (which gets available on hovering "Error" icon)
      
Import Extension Catalog
- Must be of Type 'kubernetes.io/dockercfg' or 'kubernetes.io/dockerconfigjson' (which gets available when users hover on "Information" icon associated with "Image Pull Secrets" label)

Banners
- Show a modal dialog on the login screen that must be accepted by a user before they can login (which gets available when users hover on "Information" icon associated with "Show Login Consent as a modal dialog" checkbox)

Home Links
- Paste lines of key=value or key:value into any key field for easy bulk entry (which gets available when users hover on "Information" icon associated with "Display Text" checkbox)

DaemonSet: Create
- Must be of Type `kubernetes.io/dockercfg` or `kubernetes.io/dockerconfigjson` (which gets available when users hover on "Information" icon associated with "Pull Secrets " label)

Example:

![image](https://github.com/user-attachments/assets/b0df5df3-6ffa-42c4-b4fa-90bd4808e3b2)

Remediation:

Modify the scripts associated with the additional content to ensure that it remains visible until user decides to dismiss the additional content.

**Comment by izaac** [comment]
Rancher version: `v2.11-0b081e077e830fa17b86151d21c786fb8d07c6de-head`
UI version: `master 68fb044`
Chrome version: `133.0.6943.141`
Firefox version: `135.0.1`

I validated all the sections/fields where the Information appears as described in the issue. ✅
